src/main/java/com/backbase/ct/bbfuel/configurator/MessagesConfigurator.java

```diff
@@ -27,53 +29,66 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(MessagesConfigurator.class);
     private static GlobalProperties globalProperties = GlobalProperties.getInstance();
+    private static final String USER_ADMIN = "admin";
 
     private final LoginRestClient loginRestClient;
     private final MessagesPresentationRestClient messagesPresentationRestClient;
     private String rootEntitlementsAdmin = globalProperties.getString(PROPERTY_ROOT_ENTITLEMENTS_ADMIN);
 
     public void ingestConversations(String externalUserId) {
         int randomAmount = CommonHelpers
-            .generateRandomNumberInRange(globalProperties.getInt(PROPERTY_MESSAGES_MIN),
-                globalProperties.getInt(PROPERTY_MESSAGES_MAX));
-        IntStream.range(0, randomAmount).forEach(randomNumber -> {
+                .generateRandomNumberInRange(globalProperties.getInt(PROPERTY_MESSAGES_MIN),
+                        globalProperties.getInt(PROPERTY_MESSAGES_MAX));
 
-            loginRestClient.login(externalUserId, externalUserId);
-            DraftsPostRequestBody draftsPostRequestBody = MessagesDataGenerator.generateDraftsPostRequestBody();
-            String draftId = messagesPresentationRestClient.postDraft(draftsPostRequestBody)
+        loginRestClient.login(USER_ADMIN, USER_ADMIN);
+
+        List<String> topicIds = new ArrayList<>();
+        String topicId = messagesPresentationRestClient.postTopic(MessagesDataGenerator.generateTopicPostRequestBody())
                 .then()
                 .statusCode(SC_ACCEPTED)
                 .extract()
-                .as(DraftsPostResponseBody.class)
-                .getId();
+                .path("id");
+        topicIds.add(topicId);
+
+        IntStream.range(0, randomAmount).forEach(randomNumber -> {
+
+            loginRestClient.login(externalUserId, externalUserId);
+
+            DraftsPostRequestBody draftsPostRequestBody = MessagesDataGenerator.generateDraftsPostRequestBody(topicIds);
```

Comment:
was it intentional having only one element in the list of topic ids? you might want to remove the list wrapper and pass on the bare topicId

Reply:
yes, it was intentionally. I think someone will want to have more topics for testing.

Reply:
updated PR